src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Reindex/ReindexUtilities.cs

```diff
@@ -91,30 +92,15 @@ public async Task ProcessSearchResultsAsync(SearchResult results, IReadOnlyDicti
 
         public async Task<(bool, string)> UpdateSearchParameters(IReadOnlyCollection<string> searchParameterUris, CancellationToken cancellationToken)
```

Comment:
>UpdateSearchParameters [](start = 42, length = 22)

nit: I think name is a bit misleading, it's enabling search parameters rather than updating them (update for me sound like more generic operation)